simplemma/language_detector.py

```diff
@@ -91,10 +111,13 @@ def lang_detector(
     results = sorted(myresults.items(), key=itemgetter(1), reverse=True)
     # post-processing
     if len(results) > 1:
-        # in case of ex-aequo
-        if greedy is False and results[0][1] == results[1][1]:
-            results = lang_detector(text, lang=lang, greedy=True)
-        # fallback
-        if len(results) > 1 and results[0][1] == results[1][1]:
-            return _return_default()
+        # switch unknown first item to the end
+        if results[0][0] == "unk":
```

Comment:
I think that it's good to keep this sorted.
What if the unknown result is the second and not the first?

You can simply check if the first or second result is unk. And skip that.

Reply:
Or just put the unknown proportion systematically at the end?